src/dom.js

```diff
@@ -126,7 +129,7 @@ export function closestByTag(element, tagName) {
  */
 export function elementByTag(element, tagName) {
   const elements = element.getElementsByTagName(tagName);
-  return elements.length > 0 ? elements[0] : null;
+  return elements[0] || null;
```

Comment:
@dvoytenko You benchmarked this to be faster, right?

I bet querySelector would win if there are actually more than 1 results, though.